modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java

```diff
@@ -297,20 +297,23 @@ public Components getComponents(ObjectNode obj, String location, ParseResult res
         if (obj == null) {
             return null;
         }
-        
-//        Map<String, Tag> tags = new LinkedHashMap<String,Tag>();
         List<Tag> tags = new ArrayList<>();
         for (JsonNode item : obj) {
             if (item.getNodeType().equals(JsonNodeType.OBJECT)) {
                 Tag tag = getTag((ObjectNode) item, location, result);
-                if (tag != null ) {
-                    if(tags.contains(tag.getName())) {
-                        LOGGER.warn("Duplicate tags found: "+tag.getName());
```

Comment:
Hi, The SwaggerParseResult Object has  a map of messages: https://github.com/swagger-api/swagger-parser/blob/master/modules/swagger-parser-core/src/main/java/io/swagger/v3/parser/core/models/SwaggerParseResult.java#L7

The main idea is to use that to warn the user, maybe look the way this PR implemented something similar:
https://github.com/swagger-api/swagger-parser/pull/873/commits/15a3881540ade50cd1c364aef8645491f046e6dd#diff-998f064dc6ef209e504969ef62654a84R2695